html/elements/menu.json

```diff
@@ -0,0 +1,318 @@
+          }
+        },
+        "hr_separator": {
+          "__compat": {
+            "description": "<code>&lt;hr&gt;</code> creates a separator",
+            "support": {
+              "webview_android": {
+                "version_added": false
+              },
+              "chrome": {
+                "version_added": null
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "edge_mobile": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "51"
+              },
+              "firefox_android": {
+                "version_added": "51"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "ie_mobile": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": null
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "label": {
+          "__compat": {
+            "support": {
+              "webview_android": {
+                "version_added": false
+              },
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "8"
+              },
+              "firefox_android": {
+                "version_added": true,
+                "notes": "Nested menus are not supported."
+              },
+              "ie": {
+                "version_added": false
+              },
+              "ie_mobile": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": true
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "type": {
```

Comment:
type="context" is missing.

I would nest type="context" and type="toolbar" under type, but I'm not sure type needs __compat (aka basic support). So:
- type
  - context
    - __compat
  - toolbar
    - __compat

Reply:
I nested context and toolbar; but without the basic support for type, the rendering macro is crashing, so I kept it.

Reply:
Opened https://github.com/mdn/browser-compat-data/pull/385

Reply:
@elchi3 I updated the PR to remove the basic support for type. It works locally, so this can land when #385 has landed.